test/functorch/test_vmap.py

```diff
@@ -3538,6 +3538,12 @@ def compute_A(out):
             self.opinfo_vmap_test(device, torch.float, op, check_has_batch_rule=False,
                                   postprocess_fn=compute_A)
 
+    def test_slogdet(self, device):
+        # There's no OpInfo for this
+        B = 2
+        x = torch.randn(2, 5, 5, device=device)
+        self.vmap_outplace_test(torch.slogdet, (x,), {}, (0,))
```

Comment:
Do we also want the `check_vmap_fallback` since IIUC it was working before but raising the warning

Reply:
Good catch. I forgot that vmap_outplace_test doesn't do the check_vmap_fallback